### ISSUE TITLE
you are a legend!

### ISSUE THREAD

**Issue by DaveFlashNL** [dangeredwolf/FixTweet#36]
may we have the whole story sir?

it seems you gotten a hold of the original OG domain, fxtwitter . com. can you please tell us a bit more?

**Comment by dangeredwolf** [comment]
For those who are out of the loop the TL;DR is that TwitFix shut down last May due to understandable privacy issues by some people and corresponding death threats from transphobes. So the domain was taken down and remained dormant for months. This situaton is how vxTwitter came to be. So a combination of me being frustrated at the limitations of TwitFix and its forks and the events that led to vxTwitter's creation led me to want to build something better from the ground up. After all, something like TwitFix was like a perfect use case scenario for Cloudflare Workers which I had just started to get used to when I began the project.

I know Robin who ran TwitFix. I asked for the domain. She said no, because she was worried about the whole situation that had happened before. And that was understandable, and I respected that. So I kinda put the idea on the shelf for just a little bit, but a few weeks later I actually decided to make my own. So I got a new domain name for it (pxtwitter.com) and I did exactly what I wanted to do—Write a fast, serverless, privacy-focused version of TwitFix with feature parity to TwitFix and derivatives like vxTwitter. I launched it in just over 24 hours from when I started, and it already had many features that TwitFix and derivatives don't have, like polls and media-based embed colors. Other things like multi-image and direct media linking I added in shortly after.

![2022-08-16T20;18;25 610_chrome](https://user-images.githubusercontent.com/2636265/185006895-87aeef8e-3351-4dac-ab6e-56e1863fd641.png)

My work didn't go unnoticed on Twitter by Robin however, and she had been a bit impressed by some of the things I came up with like poll support, etc. which I always thought was super dope since she was the one that made TwitFix and was the one who inspired me to makek this project in the first place. And I had talked back and forth with her about the project and stuff. I didn't ask for the domain or anything again cause I valued our relationship more than a domain name. But out of the blue, Robin DMed me simply with "All right, you've convinced me" and asked for my email--so it turns out I was getting the domain after all.

As soon as I got the domain I made an announcement on my Discord server 'cause I was hyped as hell.

![2022-08-16T20;21;30 794_DiscordCanary](https://user-images.githubusercontent.com/2636265/185007134-7bea80e6-40e5-4794-bf57-d87caffd81a6.png)

Robin actually joined my Discord server at this time. Before I switched on the domain I had some chats with some other members of my Discord server including Robin as to how we handle the name pivot—because the name pxTwitter just didn't really fit if our new domain was fxtwitter.com—So I wanted a new name for the project that was more related to the fxtwitter.com domain, but still unique from TwitFix—and FixTweet was something I came up with pretty fast that resonated with me well. And so there it was! With a new name having been pushed out to the worker I was ready to turn on fxtwitter.com for the first time in months—now powered by FixTweet's grounds-up embedder.

Even I was surprised how quickly stuff started to spring back to life when the fxtwitter.com domain was turned back on.

![2022-08-16T20;13;00 363_DiscordCanary](https://user-images.githubusercontent.com/2636265/185006358-5465ebc9-4fa5-4ec8-b05a-ed04002681bf.png)

Getting the fxtwitter.com domain has catapulted FixTweet into a position it otherwise would have probably never gotten, and I hope more and more people continue to use the new fxtwitter.com and put pressure on vxTwitter to actually add more features to its service instead of taking away.

Now, FixTweet has a free Tweet fetch API we built from scratch, using the same data we use for the FixTweet embedder but given in an easy-to-parse JSON format, and I hope I can add more functionality to our API in the future to help people make useful things without having to deal with Twitter's cumbersome API.

Recently, I have deprecated pxtwitter.com in favor of fxtwitter.com to encourage the few people still using the old domain to migrate. But I'll keep old links working for the foreseeable future.

I'd obviously love to thank Robin for the work she did in building the original TwitFix and for giving me the fxtwitter.com domain for free. And everyone that used the OG pxtwitter before we got the fxtwitter.com domain—These people are the OGs to me. And of course a thank you to everyone using fxtwitter.com today—let everyone using other embedders know that fxtwitter.com is back!
